src/helium_follower_service.erl

```diff
@@ -5,12 +5,17 @@
 %%%-------------------------------------------------------------------
 -module(helium_follower_service).
 
--behaviour(helium_follower_bhvr).
+-behaviour(helium_follower_follower_bhvr).
```

Comment:
this isn't a typo?

Reply:
unfortunately not; the proto file is `follower.proto` and the package defined in there is `helium.follower` so when grpcbox_plugin constructs this behaviour it's calling it that by joining the package name and the file name and sticking `bhvr` on the end

Reply:
i didn't see any way to override the naming convention in the config without modifying our fork of grpcbox_plugin and i didn't think these are expected to live long enough to be worth that

Reply:
OK lol